stream_alert/rule_processor/rules_engine.py

```diff
@@ -342,25 +342,24 @@ def process(self, input_payload):
         and the rule itself to determine if a match occurs.
 
         Returns:
-            list: alerts
-
-            An alert is represented as a dictionary with the following keys:
-                rule_name: the name of the triggered rule
-                payload: the StreamPayload object
-                outputs: list of outputs to send to
+            (list): First return is a list of alerts.
+                An alert is represented as a dictionary with the following keys:
+                    rule_name: the name of the triggered rule
+                    payload: the StreamPayload object
+                    outputs: list of outputs to send to
+            (list): Second return is a list of payload instance with normalized records.
```

Comment:
Same comment as above. Now that I'm looking at it, please specify that this returns a `tuple<list, list>` instead.